<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="publicKeyBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="publicKeyLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Public Key (.pem)</string>
    </property>
   </widget>
   <widget class="QLabel" name="privateKeyLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Private Key (.pem)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="privateKeyBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="aboutLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>570</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Marco Cheung @ 2022, https://github.com/marc0cheung</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>761</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
